--- index.html	2018-09-18 21:20:35.000000000 +0900
+++ index.html.a	2018-09-18 21:20:20.000000000 +0900
@@ -5,6 +5,13 @@
         <title>Preview</title>
         <link type="text/css" href="css/previm.css" rel="stylesheet" media="all" />
         <link type="text/css" href="../_/css/lib/mermaid.min.css" rel="stylesheet" media="all" />
+        <script type="text/x-mathjax-config">
+          MathJax.Hub.Config({
+            CommonHTML: { matchFontHeight: false }
+          });
+        </script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML" id="">
+        </script>
         <style>
           /* Fix #43 */
           .section,
